sensor/common/deploymentenhancer/component_test.go

```diff
@@ -82,6 +82,34 @@ func (s *ComponentTestSuite) TestExtractAndEnrichDeployments() {
 	s.Len(actual, expected)
 }
 
+func (s *ComponentTestSuite) TestEnhanceDeploymentsEmptyMessage() {
+	dQueue := make(chan *central.DeploymentEnhancementRequest, 10)
+	de := DeploymentEnhancer{
+		responsesC:       make(chan *message.ExpiringMessage),
+		deploymentsQueue: dQueue,
+		storeProvider:    s.mockStoreProvider,
+	}
+	msg := &central.DeploymentEnhancementRequest{}
+
+	actual := de.enhanceDeployments(msg)
+
+	s.Empty(actual)
+}
+
+func (s *ComponentTestSuite) TestEnhanceDeploymentsNoDeployments() {
```

Comment:
It looks like this test and the previous one could be merged into a single table-driven test.

Reply:
Done!